src/match/matchMakingFunction.ts

```diff
@@ -0,0 +1,83 @@
+import { MatchMakingQueueItem } from "./MatchMakingQueueItem";
+
+export const avg = (values: number[]) => {
+  return values.reduce((sum, current) => sum + current, 0) / values.length;
+};
+
+export const variance = (values: number[], waitingTime: number[]) => {
```

Comment:
`values` -> `rating`

Reply:
After looking at the code more, I don't think variance is a really good name. Maybe something like `MatchQuality`?